packages/component-library/draft/Kaizen/SplitButton/styles.scss

```diff
@@ -158,6 +220,22 @@ $button-width: 124px;
   }
 }
 
+.dropdownButtonDefault {
+  composes: buttonReset buttonDefault;
+}
+
+.dropdownButtonPrimary {
+  composes: buttonReset buttonPrimary;
+
+  // Making the left | right border white to separate the button with dropdown button
+  border-left-color: $white !important;
+
+  [dir="rtl"] & {
+    border-left-color: $ca-palette-seedling !important;
+    border-right-color: $white !important;
```

Comment:
Using `!important` is generally considered an anti-pattern. Just confirming if these are needed?

Reply:
i see. I couldn't find another way to override this. The thing with the primary color is that the border color is the same color (`seedling`) instead of white (https://github.com/cultureamp/kaizen-design-system/pull/120#issuecomment-553151667) but i put the white color for separating the dropwdown with the main button. If we don't like that we can just make the separator same color as the border `seedling`.
will look like this:

![image](https://user-images.githubusercontent.com/3228237/72539216-ba7ba200-385d-11ea-9c91-f94423fffe4d.png)